<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from khr-3hv_urdf.xacro             | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="khr-3hv" xmlns:xacro="http://ros.org/wiki/xacro">
  <!-- common color -->
  <!-- calculate box inertia -->
  <!-- common color -->
  <!-- calculate box inertia -->
  <!-- common color -->
  <!-- calculate box inertia -->
  <!-- common color -->
  <!-- calculate box inertia -->
  <!-- common color -->
  <!-- calculate box inertia -->
  <!-- head -->
  <joint name="base_link_to_head" type="revolute">
    <parent link="base_link"/>
    <child link="head"/>
    <origin rpy="0 0 0" xyz="0 0 0.0315"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000" lower="-1.57079637" upper="1.57079637" velocity="0.5"/>
  </joint>
  <link name="head">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0.017"/>
      <geometry>
        <box size="0.052 0.031 0.033"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0.017"/>
      <geometry>
        <box size="0.052 0.031 0.033"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0.017"/>
      <mass value="0.119"/>
      <inertia ixx="3.76139166667e-05" ixy="0" ixz="0" iyy="3.63445833333e-05" iyz="0" izz="2.03291666667e-05"/>
    </inertial>
  </link>
  <!-- base link-->
  <link name="base_link">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.129 0.105 0.062"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.129 0.105 0.062"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <mass value="0.3655"/>
      <inertia ixx="0.000623938958333" ixy="0" ixz="0" iyy="0.00084266025" iyz="0" izz="0.000452884958333"/>
    </inertial>
  </link>
  <joint name="base_link_to_body_2" type="revolute">
    <parent link="base_link"/>
    <child link="body_2"/>
    <origin rpy="0 0 0" xyz="0 0 -0.0525"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000" lower="-0.785398185" upper="0.785398185" velocity="0.5"/>
  </joint>
  <link name="body_2">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.057 0.08 0.041"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.057 0.08 0.041"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <mass value="0.3655"/>
      <inertia ixx="0.000150159583333" ixy="0" ixz="0" iyy="0.000293892458333" iyz="0" izz="0.000246133791667"/>
    </inertial>
  </link>
  <!-- left arm -->
  <joint name="base_link_to_Lshoulder" type="revolute">
    <parent link="base_link"/>
    <child link="Lshoulder"/>
    <origin rpy="0 0 0" xyz="0 0.0525 0.018"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-3.14159274" upper="1.04719758" velocity="0.5"/>
  </joint>
  <link name="Lshoulder">
    <visual>
      <origin rpy="0 0 0" xyz="0 0.023 0"/>
      <geometry>
        <box size="0.026 0.026 0.036"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0.023 0"/>
      <geometry>
        <box size="0.026 0.026 0.036"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0.023 0"/>
      <mass value="0.02975"/>
      <inertia ixx="4.88891666667e-06" ixy="0" ixz="0" iyy="3.35183333333e-06" iyz="0" izz="4.88891666667e-06"/>
    </inertial>
  </link>
  <joint name="Lshoulder_to_Lupper_arm" type="revolute">
    <parent link="Lshoulder"/>
    <child link="Lupper_arm"/>
    <origin rpy="0 0 0" xyz="0 0.023 0"/>
    <axis xyz="1 0 0"/>
    <limit effort="1000" lower="-0.0" upper="1.57079637" velocity="0.5"/>
  </joint>
  <link name="Lupper_arm">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 -0.079"/>
      <geometry>
        <box size="0.026 0.026 0.082"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 -0.079"/>
      <geometry>
        <box size="0.026 0.026 0.082"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 -0.079"/>
      <mass value="0.02975"/>
      <inertia ixx="1.83458333333e-05" ixy="0" ixz="0" iyy="3.35183333333e-06" iyz="0" izz="1.83458333333e-05"/>
    </inertial>
  </link>
  <joint name="Lupper_arm_to_Llower_arm" type="revolute">
    <parent link="Lupper_arm"/>
    <child link="Llower_arm"/>
    <origin rpy="0 0 0" xyz="0.013 0 -0.125"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-2.61799395" upper="0.0" velocity="0.5"/>
  </joint>
  <link name="Llower_arm">
    <visual>
      <origin rpy="0 0 0" xyz="-0.013 0 -0.033"/>
      <geometry>
        <box size="0.026 0.026 0.056"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="-0.013 0 -0.033"/>
      <geometry>
        <box size="0.026 0.026 0.056"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="-0.013 0 -0.033"/>
      <mass value="0.051"/>
      <inertia ixx="1.6201e-05" ixy="0" ixz="0" iyy="5.746e-06" iyz="0" izz="1.6201e-05"/>
    </inertial>
  </link>
  <!-- right arm -->
  <joint name="base_link_to_Rshoulder" type="revolute">
    <parent link="base_link"/>
    <child link="Rshoulder"/>
    <origin rpy="0 0 0" xyz="0 -0.0525 0.018"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-3.14159274" upper="1.04719758" velocity="0.5"/>
  </joint>
  <link name="Rshoulder">
    <visual>
      <origin rpy="0 0 0" xyz="0 -0.023 0"/>
      <geometry>
        <box size="0.026 0.026 0.036"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 -0.023 0"/>
      <geometry>
        <box size="0.026 0.026 0.036"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 -0.023 0"/>
      <mass value="0.02975"/>
      <inertia ixx="4.88891666667e-06" ixy="0" ixz="0" iyy="3.35183333333e-06" iyz="0" izz="4.88891666667e-06"/>
    </inertial>
  </link>
  <joint name="Rshoulder_to_Rupper_arm" type="revolute">
    <parent link="Rshoulder"/>
    <child link="Rupper_arm"/>
    <origin rpy="0 0 0" xyz="0 -0.023 0"/>
    <axis xyz="1 0 0"/>
    <limit effort="1000" lower="-1.57079637" upper="0.0" velocity="0.5"/>
  </joint>
  <link name="Rupper_arm">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 -0.079"/>
      <geometry>
        <box size="0.026 0.026 0.082"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 -0.079"/>
      <geometry>
        <box size="0.026 0.026 0.082"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 -0.079"/>
      <mass value="0.02975"/>
      <inertia ixx="1.83458333333e-05" ixy="0" ixz="0" iyy="3.35183333333e-06" iyz="0" izz="1.83458333333e-05"/>
    </inertial>
  </link>
  <joint name="Rupper_arm_to_Rlower_arm" type="revolute">
    <parent link="Rupper_arm"/>
    <child link="Rlower_arm"/>
    <origin rpy="0 0 0" xyz="0.013 0 -0.125"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-2.61799395" upper="0.0" velocity="0.5"/>
  </joint>
  <link name="Rlower_arm">
    <visual>
      <origin rpy="0 0 0" xyz="-0.013 0 -0.033"/>
      <geometry>
        <box size="0.026 0.026 0.056"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="-0.013 0 -0.033"/>
      <geometry>
        <box size="0.026 0.026 0.056"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="-0.013 0 -0.033"/>
      <mass value="0.051"/>
      <inertia ixx="1.6201e-05" ixy="0" ixz="0" iyy="5.746e-06" iyz="0" izz="1.6201e-05"/>
    </inertial>
  </link>
  <!-- left leg -->
  <joint name="body_2_to_Lhips" type="revolute">
    <parent link="body_2"/>
    <child link="Lhips"/>
    <origin rpy="0 0 0" xyz="0 0.03 -0.0255"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000" lower="-0.52359879" upper="0.785398185" velocity="0.5"/>
  </joint>
  <link name="Lhips">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.03 0.03 0.0039"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.03 0.03 0.0039"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <mass value="0.03944"/>
      <inertia ixx="3.0079902e-06" ixy="0" ixz="0" iyy="5.916e-06" iyz="0" izz="3.0079902e-06"/>
    </inertial>
  </link>
  <joint name="Lhips_to_Lthigh" type="revolute">
    <parent link="Lhips"/>
    <child link="Lthigh"/>
    <origin rpy="0 0 0" xyz="0 0 -0.030"/>
    <axis xyz="1 0 0"/>
    <limit effort="1000" lower="-0.52359879" upper="0.785398185" velocity="0.5"/>
  </joint>
  <link name="Lthigh">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.03 0.03 0.026"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.03 0.03 0.026"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <mass value="0.03944"/>
      <inertia ixx="5.17978666667e-06" ixy="0" ixz="0" iyy="5.916e-06" iyz="0" izz="5.17978666667e-06"/>
    </inertial>
  </link>
  <joint name="Lthigh_to_Lknee" type="revolute">
    <parent link="Lthigh"/>
    <child link="Lknee"/>
    <origin rpy="0 0 0" xyz="0 0 -0.046"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-2.09439516" upper="0.785398185" velocity="0.5"/>
  </joint>
  <link name="Lknee">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 -0.034"/>
      <geometry>
        <box size="0.03 0.03 0.088"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 -0.034"/>
      <geometry>
        <box size="0.03 0.03 0.088"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 -0.034"/>
      <mass value="0.11832"/>
      <inertia ixx="8.522984e-05" ixy="0" ixz="0" iyy="1.7748e-05" iyz="0" izz="8.522984e-05"/>
    </inertial>
  </link>
  <joint name="Lknee_to_Lshin" type="revolute">
    <parent link="Lknee"/>
    <child link="Lshin"/>
    <origin rpy="0 0 0" xyz="-0.0205 0 -0.0885"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-0.0" upper="2.61799395" velocity="0.5"/>
  </joint>
  <link name="Lshin">
    <visual>
      <origin rpy="0 0 0" xyz="0.0205 0 -0.024"/>
      <geometry>
        <box size="0.03 0.03 0.047"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0.0205 0 -0.024"/>
      <geometry>
        <box size="0.03 0.03 0.047"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0.0205 0 -0.024"/>
      <mass value="0.04692"/>
      <inertia ixx="1.215619e-05" ixy="0" ixz="0" iyy="7.038e-06" iyz="0" izz="1.215619e-05"/>
    </inertial>
  </link>
  <joint name="Lshin_to_Lankle" type="revolute">
    <parent link="Lshin"/>
    <child link="Lankle"/>
    <origin rpy="0 0 0" xyz="0.0205 0 -0.0319"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-1.57079637" upper="1.04719758" velocity="0.5"/>
  </joint>
  <link name="Lankle">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 -0.037"/>
      <geometry>
        <box size="0.03 0.03 0.02"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 -0.037"/>
      <geometry>
        <box size="0.03 0.03 0.02"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 -0.037"/>
      <mass value="0.04692"/>
      <inertia ixx="5.083e-06" ixy="0" ixz="0" iyy="7.038e-06" iyz="0" izz="5.083e-06"/>
    </inertial>
  </link>
  <joint name="Lankle_to_Lfoot" type="revolute">
    <parent link="Lankle"/>
    <child link="Lfoot"/>
    <origin rpy="0 0 0" xyz="0 0 -0.037"/>
    <axis xyz="1 0 0"/>
    <limit effort="1000" lower="-0.785398185" upper="0.785398185" velocity="0.5"/>
  </joint>
  <link name="Lfoot">
    <visual>
      <origin rpy="0 0 0" xyz="0 0.0185 -0.0315"/>
      <geometry>
        <box size="0.122 0.078 0.01"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0.0185 -0.0315"/>
      <geometry>
        <box size="0.122 0.078 0.01"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0.0185 -0.0315"/>
      <mass value="0.02346"/>
      <inertia ixx="2.929372e-05" ixy="0" ixz="0" iyy="4.099244e-05" iyz="0" izz="1.208972e-05"/>
    </inertial>
  </link>
  <joint name="Lfsensor1_joint" type="fixed">
    <origin rpy="0 0 0" xyz="0 0 -0.0365"/>
    <parent link="Lfoot"/>
    <child link="Lfsensor1"/>
  </joint>
  <link name="Lfsensor1">
 		</link>
  <sensor name="Lfsensor1_sensor" update_rate="20">
    <origin rpy="0 0 0" xyz="0 0 -0.0365"/>
    <parent link="Lfoot"/>
  </sensor>
  <!-- right leg -->
  <joint name="body_2_to_Rhips" type="revolute">
    <parent link="body_2"/>
    <child link="Rhips"/>
    <origin rpy="0 0 0" xyz="0 -0.03 -0.0255"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000" lower="-0.785398185" upper="0.52359879" velocity="0.5"/>
  </joint>
  <link name="Rhips">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.03 0.03 0.0039"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.03 0.03 0.0039"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <mass value="0.03944"/>
      <inertia ixx="3.0079902e-06" ixy="0" ixz="0" iyy="5.916e-06" iyz="0" izz="3.0079902e-06"/>
    </inertial>
  </link>
  <joint name="Rhips_to_Rthigh" type="revolute">
    <parent link="Rhips"/>
    <child link="Rthigh"/>
    <origin rpy="0 0 0" xyz="0 0 -0.030"/>
    <axis xyz="1 0 0"/>
    <limit effort="1000" lower="-0.785398185" upper="0.52359879" velocity="0.5"/>
  </joint>
  <link name="Rthigh">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.03 0.03 0.026"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.03 0.03 0.026"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <mass value="0.03944"/>
      <inertia ixx="5.17978666667e-06" ixy="0" ixz="0" iyy="5.916e-06" iyz="0" izz="5.17978666667e-06"/>
    </inertial>
  </link>
  <joint name="Rthigh_to_Rknee" type="revolute">
    <parent link="Rthigh"/>
    <child link="Rknee"/>
    <origin rpy="0 0 0" xyz="0 0 -0.046"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-2.09439516" upper="0.785398185" velocity="0.5"/>
  </joint>
  <link name="Rknee">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 -0.034"/>
      <geometry>
        <box size="0.03 0.03 0.088"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 -0.034"/>
      <geometry>
        <box size="0.03 0.03 0.088"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 -0.034"/>
      <mass value="0.11832"/>
      <inertia ixx="8.522984e-05" ixy="0" ixz="0" iyy="1.7748e-05" iyz="0" izz="8.522984e-05"/>
    </inertial>
  </link>
  <joint name="Rknee_to_Rshin" type="revolute">
    <parent link="Rknee"/>
    <child link="Rshin"/>
    <origin rpy="0 0 0" xyz="-0.0205 0 -0.0885"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-0.0" upper="2.61799395" velocity="0.5"/>
  </joint>
  <link name="Rshin">
    <visual>
      <origin rpy="0 0 0" xyz="0.0205 0 -0.024"/>
      <geometry>
        <box size="0.03 0.03 0.047"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0.0205 0 -0.024"/>
      <geometry>
        <box size="0.03 0.03 0.047"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0.0205 0 -0.024"/>
      <mass value="0.04692"/>
      <inertia ixx="1.215619e-05" ixy="0" ixz="0" iyy="7.038e-06" iyz="0" izz="1.215619e-05"/>
    </inertial>
  </link>
  <joint name="Rshin_to_Rankle" type="revolute">
    <parent link="Rshin"/>
    <child link="Rankle"/>
    <origin rpy="0 0 0" xyz="0.0205 0 -0.0319"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000" lower="-1.57079637" upper="1.04719758" velocity="0.5"/>
  </joint>
  <link name="Rankle">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 -0.037"/>
      <geometry>
        <box size="0.03 0.03 0.02"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 -0.037"/>
      <geometry>
        <box size="0.03 0.03 0.02"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 -0.037"/>
      <mass value="0.04692"/>
      <inertia ixx="5.083e-06" ixy="0" ixz="0" iyy="7.038e-06" iyz="0" izz="5.083e-06"/>
    </inertial>
  </link>
  <joint name="Rankle_to_Rfoot" type="revolute">
    <parent link="Rankle"/>
    <child link="Rfoot"/>
    <origin rpy="0 0 0" xyz="0 0 -0.037"/>
    <axis xyz="1 0 0"/>
    <limit effort="1000" lower="-0.785398185" upper="0.785398185" velocity="0.5"/>
  </joint>
  <link name="Rfoot">
    <visual>
      <origin rpy="0 0 0" xyz="0 -0.0185 -0.0315"/>
      <geometry>
        <box size="0.122 0.078 0.01"/>
      </geometry>
      <material name="green">
        <color rgba="0 1 0 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 -0.0185 -0.0315"/>
      <geometry>
        <box size="0.122 0.078 0.01"/>
      </geometry>
    </collision>
    <inertial>
      <origin rpy="0 0 0" xyz="0 -0.0185 -0.0315"/>
      <mass value="0.02346"/>
      <inertia ixx="2.929372e-05" ixy="0" ixz="0" iyy="4.099244e-05" iyz="0" izz="1.208972e-05"/>
    </inertial>
  </link>
  <joint name="Rfsensor1_joint" type="fixed">
    <origin rpy="0 0 0" xyz="0 0 -0.0365"/>
    <parent link="Rfoot"/>
    <child link="Rfsensor1"/>
  </joint>
  <link name="Rfsensor1">
 		</link>
  <sensor name="Rfsensor1_sensor" update_rate="20">
    <origin rpy="0 0 0" xyz="0 0 -0.0365"/>
    <parent link="Rfoot"/>
  </sensor>
</robot>

